console/src/main/scripts/bower.json

```diff
@@ -37,7 +37,9 @@
     "keycloak": "1.3.1",
     "angular-truncate": "sparkalow/angular-truncate",
     "angular-smart-truncate": "1.0.2",
-    "angular-wizard": "0.5.5"
+    "angular-wizard": "0.5.5",
+    "ng-clip": "~0.2.6",
```

Comment:
Just FYI, because of being bitten by changing software packages we have been inclined to use fixed versions and not allow patch versions '~'.